assets/package.json

```diff
@@ -0,0 +1,140 @@
+{
+	"scripts": {
+		"build": "npm run build:production",
+		"prebuild:dev": "npm run remove-dist",
+		"build:dev": "webpack --mode=development --devtool cheap-module-eval-source-map --output-pathinfo",
+		"prebuild:production": "npm run remove-dist",
+		"build:production": "webpack --mode=production",
+		"build:test": "npm run build:production -- --include-tests",
+		"watch": "npm run build:dev -- --watch",
+		"remove-dist": "rimraf ../dist"
+	},
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
+	],
+	"dependencies": {
+		"@lit-labs/react": "^1.2.0",
+		"@material-ui/core": "^4.11.4",
+		"@material/button": "^2.3.0",
+		"@material/checkbox": "^2.3.0",
+		"@material/dialog": "^2.3.0",
+		"@material/form-field": "^2.3.0",
+		"@material/layout-grid": "^0.41.0",
+		"@material/linear-progress": "^1.1.0",
+		"@material/list": "^2.3.0",
+		"@material/menu": "^2.3.0",
+		"@material/radio": "^2.3.0",
+		"@material/react-card": "^0.15.0",
+		"@material/react-chips": "^0.15.0",
+		"@material/react-dialog": "^0.15.0",
+		"@material/react-select": "^0.15.0",
+		"@material/react-tab-bar": "^0.15.0",
+		"@material/react-text-field": "^0.15.0",
+		"@material/ripple": "^2.3.0",
+		"@material/select": "^2.3.1",
+		"@material/switch": "^2.3.0",
+		"@material/textfield": "^2.3.1",
+		"@material/theme": "^1.1.0",
+		"@material/web": "^1.0.0-pre.9",
+		"@popperjs/core": "^2.9.2",
+		"@reach/combobox": "^0.10.5",
+		"@react-hook/merged-ref": "^1.3.0",
+		"@react-hook/window-size": "^3.0.7",
```

Comment:
One more candidate to be removed. We don't use it right?

Reply:
Yeah, I was being a bit cautious about removing packages in this PR, but I do think we're not using those that you've flagged. I've removed this one.

Reply:
Hmm, you're right that we aren't using it directly. However we are using its dependencies `@react-hook/event` and `@react-hook/throttle`, which we didn't have installed as direct dependencies. I've replaced `@react-hook/window-size` with `@react-hook/event` and `@react-hook/throttle`.

https://github.com/google/site-kit-wp/blob/aafa3a6dfba25bcc3dfe042e043b01d726daa0f0/package-lock.json#L8721-L8733

https://github.com/google/site-kit-wp/blob/aafa3a6dfba25bcc3dfe042e043b01d726daa0f0/assets/js/hooks/useWindowSize.js#L25-L49

Reply:
Awesome! Thanks.